systemtests/CMakeLists.txt

```diff
@@ -216,23 +212,34 @@ foreach(TEST_NAME ${SYSTEM_TESTS})
   math(EXPR  fd_port "${BASEPORT} + 1")
   math(EXPR  sd_port "${BASEPORT} + 2")
   math(EXPR sd2_port "${BASEPORT} + 3")
+  math(EXPR php_port "${BASEPORT} + 4")
 
 
   # set(DEFAULT_DB_TYPE )
   set(archivedir ${current_test_directory}/storage)
   set(confdir ${current_test_directory}/etc/bareos)
 
+  # set webui conf dir
+  set(webuiconfdir ${current_test_directory}/etc/bareos-webui)
```

Comment:
Please check if the webui specific lines can move to the webui conditional part below.

Reply:
Suggested change applied.